### PR TITLE
1inch Wallet: Deploy environment on all networks

### PR DESCRIPTION
**Pull request by dmf7z**
This PR is to deploy fee collector environment for 1inch Wallet on Ethereum, Avalanche, Arbitrum, Optimism, Fantom, Gnosis, BNB, Polygon, and Base.

### CODE REVIEW

File: packages/users/1inch/wallet-fee-collector/scripts/2023111400-environment-deploy/input.arbitrum.ts [hunk 1]
```diff
@@ -0,0 +1,469 @@
...
+const STANDARD_GAS_PRICE_LIMIT = 10e9
+const TX_COST_LIMIT_PCT = fp(0.02) // 2%
+const QUOTA = fp(0.0025)
+const MIN_WINDOW_GAS = QUOTA
+const MAX_WINDOW_GAS = QUOTA.mul(10)
+
+//Config - Withdrawer Timelock
+const BRIDGER_TIMELOCK_MODE = TIMELOCK_MODE.ON_LAST_DAY //SECONDS
+const BRIDGER_TIMELOCK_FREQUENCY = 1 //1 month
+const BRIDGER_TIMELOCK_ALLOWED_AT = 1701345600 //12:00hs UTC
```
Comment (ragonzal) [inline packages/users/1inch/wallet-fee-collector/scripts/2023111400-environment-deploy/input.arbitrum.ts:43]: I would deploy with the next period, given that this epoch is tomorrow.

File: packages/users/1inch/wallet-fee-collector/scripts/2023111400-environment-deploy/input.mainnet.ts [hunk 2]
```diff
@@ -0,0 +1,729 @@
...
+          tokenIndexConfig: {
+            acceptanceType: 1, //Allow list
+            tokens: [USDC],
+          },
+        },
+      },
+    },
+    //Depositor: for manual transfers and testing purposes
+    {
+      from: DEPLOYER,
```
Comment (ragonzal) [inline packages/users/1inch/wallet-fee-collector/scripts/2023111400-environment-deploy/input.mainnet.ts:229]: No threshold?

**Review by ragonzal — APPROVED** [review]
Everything looks OK, all flows start and end as expected.
I do have to check again quotas and gas prices.

I would update the timelocks given that they are almost expired.
Shouldn't we add in mainnet (as a safety measure) some threshold for the depositors? Just to not drain funds in the relayer because we are trying to collect some shady/spam token.

File: packages/users/1inch/wallet-fee-collector/scripts/2023111400-environment-deploy/input.mainnet.ts (hunk 2 above)
Comment (dmf7z) [reply packages/users/1inch/wallet-fee-collector/scripts/2023111400-environment-deploy/input.mainnet.ts:229]: No need, it is only for testing purposes.

**Pull request merged**